src/containers/Header/index.js

```diff
@@ -65,14 +65,14 @@ export function Header(props) {
 
 			}
 
-				{/* toggler */}
-				<div className={`header-props-toggler mt-1 mr-3 p-0 ${headerProperties && 'header-props-open' }`} onClick={() => setHeaderProperties(!headerProperties)}>
-					<i className="cil-chevron-bottom"></i>
-				</div>
+			{/* toggler */}
+			<div className={`header-props-toggler mr-3 p-0 ${headerProperties ? 'header-props-open' : '' }`} onClick={() => setHeaderProperties(!headerProperties)}>
```

Comment:
@petrKavulok what was wrong with this className condition?

```
headerProperties && 'header-props-open'
```

Reply:
it would for some reason return `class="header-props-toggler mr-3 p-0 false"`

<img width="538" alt="Snímek obrazovky 2022-06-29 v 20 57 36" src="https://user-images.githubusercontent.com/79644454/176611442-6a5b7d32-8cc0-4150-a084-e23b8d8bca69.png">